### ISSUE TITLE
Add support for removing components and projects

### ISSUE THREAD

**Issue by nijel** [WeblateOrg/weblate#2221]
Admins should be able to remove translation components and projects from Weblate UI (especially after #2220 is implemented). This should however provide some additional safety to avoid accidental project removal. Options which come to my mind (without any preference and without saying that this is complete list of choices which should be considered):

* Typing project/component name in the dialog, this is how GitHub does it
* Email confirmation similarly like we do with account removal

**Comment by comradekingu** [comment]
This would help rid project lists of duplicated resources.

**Comment by nijel** [comment]
Thank you for your report, the issue you have reported has just been fixed. 

* In case you see problem with the fix, please comment on this issue. 
* In case you see similar problem, please open separate issue.
* If you are happy with the outcome, consider supporting Weblate by [donating](https://weblate.org/donate/).
